### PR TITLE
feat(Contacts): Multiple Emails in a Contact

### PR DESCRIPTION
**Pull request by hrwX**
[#8145](https://github.com/frappe/frappe/pull/8145)

### CODE REVIEW

File: erpnext/communication/doctype/call_log/call_log.py [hunk 1]
```diff
@@ -73,6 +73,10 @@ def set_caller_information(doc, state):
 	# contact_name or lead_name
 	display_name_field = '{}_name'.format(fieldname)
 
+	# Contact now has all the nos saved in child table
+	if doc.doctype == 'Contact':
+		numbers = [nos.phone for nos in doc.phone_nos]
```
Comment (surajshetty3416) [inline erpnext/communication/doctype/call_log/call_log.py]:
```suggestion
		numbers = [d.phone for d in doc.phone_nos]
```

File: erpnext/crm/doctype/opportunity/test_opportunity.py [hunk 2]
```diff
@@ -45,15 +45,16 @@ def test_make_new_lead_if_required(self):
 
 		# create new customer and create new contact against 'new.opportunity@example.com'
 		customer = make_customer(opp_doc.party_name).insert(ignore_permissions=True)
-		frappe.get_doc({
+		d = frappe.get_doc({
```
Comment (surajshetty3416) [inline erpnext/crm/doctype/opportunity/test_opportunity.py]:
```suggestion
		contact = frappe.get_doc({
```

File: erpnext/public/js/templates/contact_list.html [hunk 3]
```diff
@@ -14,20 +14,33 @@
...
+			{% endif %}
+			{% if(contact_list[i].phone_nos) { %}
+				{% for(var j=0, k=contact_list[i].phone_nos.length; j<k; j++) { %}
+					{%= __("Phone ") %}: {%= contact_list[i].phone_nos[j].phone %}<br>
+				{% } %}
+			{% endif %}
+		</p>
+		<p>
+			{% if(contact_list[i].email_id) { %}
+				{%= __("Email ") %}: {%= contact_list[i].email_id %}<span class="text-muted"> ({%= __("Primary") %})</span><br>
```
Comment (surajshetty3416) [inline erpnext/public/js/templates/contact_list.html]:
```suggestion
				{%= __("Email") %}: {%= contact_list[i].email_id %}<span class="text-muted"> ({%= __("Primary") %})</span><br>
```

Not good for translation

translation for `__('Email')` will not be equal to `__('Email ')`

please fix `__('Phone ')` as well

**Review by surajshetty3416 — CHANGES_REQUESTED** [review]

**Review by surajshetty3416 — DISMISSED** [review]

**Comment by surajshetty3416** [comment]
Fix [this method](https://github.com/frappe/erpnext/blob/develop/erpnext/accounts/doctype/sales_invoice/pos.py#L224) as well

**Review by surajshetty3416 — APPROVED** [review]

**Comment by muchai** [comment]
There's a minor bug on this, handling apostrophes (') in emails: https://github.com/frappe/erpnext/issues/18927

**Pull request merged**